nfs.yml

```
@@ -3,7 +3,7 @@
   collections:
     - kubernetes.core
   vars:
-    storageClass: ebs
+    persistence_storage_class: eb
```

eb is a bit hard to guess. How about just `block_storage` instead? or even `generic_block_storage`
```suggestion
    persistence_storage_class: block_storage
```